bmorph/util/mizuroute_utils.py

```diff
@@ -393,7 +399,7 @@ def map_segs_topology(routed: xr.Dataset, topology: xr.Dataset):
         The input dataset routed updated with the topological data.
     """
     routed = routed.sel(seg=topology['seg'])
-    #routed['contributing_area'] = topology['Contrib_Area']
+    routed['contributing_area'] = topology['Contrib_Area']
```

Comment:
I actually don't think the contributing area is needed - maybe we delete it?

Reply:
I need it for doing the area weighted over upstream sum for IBC. I can resource it outside of the code, but do need that variable one way or another. I think the elevation can be gotten rid of though